src/lib389/lib389/cli_conf/plugins/automember.py

```diff
@@ -156,12 +156,21 @@ def fixup(inst, basedn, log, args):
     if not plugin.status():
         log.error("'%s' is disabled. Rebuild membership task can't be executed" % plugin.rdn)
     fixup_task = plugin.fixup(args.DN, args.filter)
-    fixup_task.wait()
-    exitcode = fixup_task.get_exit_code()
-    if exitcode != 0:
-        log.error('Rebuild membership task for %s has failed. Please, check logs')
+    if args.wait:
+        log.info(f'Waiting for fixup task "{fixup_task.dn}" to complete.  You can safely exit by pressing Control C ...')
+        fixup_task.wait(timeout=None)
+        exitcode = fixup_task.get_exit_code()
+        if exitcode != 0:
+            log.error(f'Rebuild membership task "{fixup_task.dn}" for {args.DN} has failed (error {exitcode}). Please, check logs')
+        else:
+            log.info('Fixup task successfully completed')
     else:
-        log.info('Successfully added task entry')
+        log.info(f'Successfully added task entry: {fixup_task.dn}')
```

Comment:
this needs to say that the task isn't completed yet, but that the task is running in the background, as well as how the user can cehck it's status.

Reply:
done